Sources/Action/AppKitExtensions/NSButton+Action.swift

```diff
@@ -0,0 +1,55 @@
+import Cocoa
+import RxSwift
+import RxCocoa
+import ObjectiveC
+
+public extension Reactive where Base: NSButton {
```

Comment:
I'm not a huge fan of this repetition... What about using OS-specific button aliases ? 

e.g. something like

```swift
#if os(iOS)
    import UIKit
    typealias OSButton = UIButton
#elseif os(macOS)
    import Cocoa
    import AppKit
    typealias OSButton = NSButton
#endif

extension Reactive where Base: OSButton {

}
```

Makes even more sense due to the fact it seems this is already done for tvOS: https://github.com/fpillet/Action/blob/896866a30c879089e740d2a94b29bca62cc9eb53/Sources/Action/UIKitExtensions/UIButton%2BRx.swift#L36

Reply:
iOS and tvOS have UIButton in common. macOS NSButton has some code differences -- it only have one target, not multiple, so you'll see the code is not the same

Reply:
Yeah for sure, but I still think there is room for condition? The differences, I think, are relatively minor IIRC

Reply:
There are plenty other macOS controls that don't exist on iOS and we'll have to add support for (I'm currently using NSStepper and NSPopUpButton a lot and would eventually like to include Action support for them as well). I think it wouldn't make sense to share just one class, it's cleaner to have separate code bases because most of the controls are too diverging

Reply:
Alright, I definitely don't have a deep enough understanding of macOS to feel very religious about this :) If it makes sense for you let's go with it. There are some styling issues at the bottom (indent and excess void) that could be worth addressing